lib/Backend/SccLiveness.cpp

```diff
@@ -399,18 +399,7 @@ SCCLiveness::ProcessSrc(IR::Opnd *src, IR::Instr *instr)
             {
                 lifetime = this->InsertLifetime(stackSym, reg, this->func->m_headInstr->m_next);
                 lifetime->region = this->curRegion;
-                lifetime->isFloat = symOpnd->IsFloat();
-                lifetime->isSimd128F4 = symOpnd->IsSimd128F4();
-                lifetime->isSimd128I4 = symOpnd->IsSimd128I4();
-                lifetime->isSimd128I8 = symOpnd->IsSimd128I8();
-                lifetime->isSimd128I16 = symOpnd->IsSimd128I16();
-                lifetime->isSimd128U4 = symOpnd->IsSimd128U4();
-                lifetime->isSimd128U8 = symOpnd->IsSimd128U8();
-                lifetime->isSimd128U16 = symOpnd->IsSimd128U16();
-                lifetime->isSimd128B4 = symOpnd->IsSimd128B4();
-                lifetime->isSimd128B8 = symOpnd->IsSimd128B8();
-                lifetime->isSimd128B16 = symOpnd->IsSimd128B16();
-                lifetime->isSimd128D2 = symOpnd->IsSimd128D2();
+                lifetime->isFloat = symOpnd->IsFloat() || symOpnd->IsSimd128();
```

Comment:
>lifetime->isFloat = symOpnd->IsFloat() || symOpnd->IsSimd128(); [](start = 16, length = 63)

Can we move this (and region) to the constructor?  I hate the fact that they are separate from everything else.